sources/packages/backend/apps/api/src/cra-integration/cra-integration.models.ts

```diff
@@ -1,10 +1,6 @@
 import { CRAResponseStatusRecord } from "./cra-files/cra-response-status-record";
 import { CRAResponseTotalIncomeRecord } from "./cra-files/cra-response-total-income-record";
 
-export const DATE_FORMAT = "YYYYMMDD";
```

Comment:
Even though these constants are the same they were defined at the specific integration level for the purpose to allow the different integration to be changed without possibly affecting the others. Are we changing this behavior in any way here?

Reply:
oh....but in some files, it was imported from another integration folder, eg, 
![image](https://user-images.githubusercontent.com/77353155/203660663-950448c0-2920-44d1-927c-5a4e41299984.png)

Reply:
nevermind. I got it. reverting

Reply:
updated